uplink/src/lib.rs

```diff
@@ -450,7 +385,91 @@ impl Uplink {
             })
         });
 
-        Ok(bridge_tx)
+        // Bridge thread to batch data and redicet actions
+        thread::spawn(|| {
+            let rt = tokio::runtime::Builder::new_current_thread()
+                .thread_name("bridge")
+                .enable_time()
+                .build()
+                .unwrap();
+
+            rt.block_on(async move {
+                if let Err(e) = bridge.start().await {
+                    error!("Bridge stopped!! Error = {:?}", e);
+                }
+            })
+        });
+
+        Ok(())
+    }
+
+    pub fn construct_builtins(&mut self, bridge: &mut Bridge) -> Result<BuiltIns, Error> {
+        let bridge_tx = bridge.tx();
```

Comment:
Are there any changes here apart from moving the code into a separate function? Please separate refactor and design change PRs l

Reply:
Have created the concept of `BuiltIns` to make the code readable, we construct bridge, attach it to the builtins, run the bridge on a background threads and finally run the built-ins